scripts/!mods_preload/msu_serialization_system.nut

```diff
@@ -0,0 +1,73 @@
+local gt = this.getroottable();
+
+gt.MSU.setupSerializationSystem <- function ()
+{
+	gt.MSU.Serialization <- {
+		Mods = [],
+
+		function registerMod( _mod, _ver )
+		{
+			if (typeof _mod != "string" || typeof _ver != "string")
```

Comment:
Any reason this isn't structured as ` a == 'string' && b == 'string'`? It would be much more logical that way imo than two inverses. with line 17 switched with the 3 in here obviously.

Reply:
I want both the `_mod` i.e. mod name **and** the `_ver` to be strings. `&&` doesn't solve that. `&&` only returns false if both of them are not strings, but returns true when one is a string and the other isn't. What I want is `||` as it returns true only when both are strings.